nav2_bt_navigator/behavior_trees/navigate_through_poses_w_replanning_and_recovery.xml

```diff
@@ -14,7 +14,7 @@
         <RateController hz="0.333">
           <RecoveryNode number_of_retries="1" name="ComputePathThroughPoses">
             <ReactiveSequence>
-              <RemovePassedGoals input_goals="{goals}" output_goals="{goals}" radius="0.7"/>
+              <RemovePassedGoals input_goals="{goals}" output_goals="{goals}" radius="0.7" waypoint_statuses_id="{waypoint_statuses_id}" />
```

Comment:
I think this should be 'waypoint_statuses', analog to the path and goal/goals handling.